src/analyzer.rs

```diff
@@ -289,19 +289,61 @@ impl<'run, 'src> Analyzer<'run, 'src> {
     Ok(())
   }
 
-  fn resolve_alias(
-    recipes: &Table<'src, Rc<Recipe<'src>>>,
-    alias: Alias<'src, Name<'src>>,
+  fn resolve_alias<'a>(
+    modules: &'a Table<'src, Justfile<'src>>,
+    recipes: &'a Table<'src, Rc<Recipe<'src>>>,
+    alias: Alias<'src, Namepath<'src>>,
   ) -> CompileResult<'src, Alias<'src>> {
+    let Alias {
+      attributes,
+      name,
+      target,
+    } = alias;
+    let target_str = format!("{target}");
+    let mut path = target.into_inner();
+    let target = path
+      .pop()
+      .expect("Internal error: Alias target path can't be empty");
+
+    let alias = Alias {
```

Comment:
I think that `resolve` can be changed so that it's implemented for the correct type, so we don't need to take apart and put the alias back together again.

Reply:
Done